src/test/java/org/kie/processmigration/service/KieServiceImplTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.processmigration.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.tomakehurst.wiremock.client.CountMatchingStrategy;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.kie.processmigration.test.Profiles;
+import org.kie.processmigration.test.MockKieServerLifecycleManager;
+import org.kie.processmigration.test.TestKieServer;
+import org.kie.processmigration.model.KieServerConfig;
+import org.kie.processmigration.model.ProcessInfo;
+import org.kie.processmigration.model.ProcessRef;
+import org.kie.processmigration.model.RunningInstance;
+import org.kie.processmigration.model.exceptions.InvalidKieServerException;
+import org.kie.processmigration.model.exceptions.ProcessDefinitionNotFoundException;
+
+import javax.inject.Inject;
+import java.util.*;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.getRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+@QuarkusTest
+@TestProfile(Profiles.MockKieServerProfile.class)
+class KieServiceImplTest {
+
+    private static final String RETRY = "retry";
+    private static final Collection<String> SUCCESS = List.of("kie-server-1", "kie-server-2");
+    private static final Collection<String> FAILED = List.of("not-found", "unauthorized", "forbidden", RETRY);
+
+    @TestKieServer
+    MockKieServerLifecycleManager mockKieServer;
+
+    @Inject
+    KieService kieService;
+
+    @AfterEach
+    void resetServer() {
+        mockKieServer.reset();
+    }
+
+    @Test
+    void testRetryErroredConfigs() throws InterruptedException, JsonProcessingException {
```

Comment:
`JsonProcessingException` not thrown
```suggestion
    void testRetryErroredConfigs() throws InterruptedException {
```

Reply:
@ruromero - please see above comment